relay-profiling/src/profile_version.rs

```diff
@@ -0,0 +1,11 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone, Default, PartialEq, Eq)]
+pub enum Version {
```

Comment:
Let's keep a specific `Version` enum for each type of profile, inside their respective module.